### ISSUE TITLE
Improve documentation around `connection_status` / others?

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#1851]
## The Issue
It's not clear what connection statuses are possible in the SDK. 

Another area to check out is the list of available languages allowed for publish. 

We should make sure to reference these values in the API documentation, directly or via a link. 

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

### Acceptance Criteria
1.
2.
3.

### Definition of Done
- [ ]  Tested against acceptance criteria
- [ ] Tested against the assumptions of user story
- [ ] The project builds without errors
- [ ] Unit tests are written and passing
- [ ] Tests on devices/browsers listed in the issue have passed
- [ ] QA performed & issues resolved
- [ ] Refactoring completed
- [ ] Any configuration or build changes documented
- [ ] Documentation updated
- [ ] Peer Code Review performed

**Comment by patrickkeane** [comment]
Considering only the first part of the issue, "possible connection statuses in the SDK", there are two possible connection status codes, shown below.  The second part of the issue, "Another area to check out is the list of available languages allowed for publish." may need its own issue, as they're not related.
```
CONNECTION_STATUS_CONNECTED = 'connected'
CONNECTION_STATUS_NETWORK = 'network_connection'
```

These are described in connection status message as:
```
CONNECTION_STATUS_CONNECTED: 'No connection problems detected',
CONNECTION_STATUS_NETWORK: "Your internet connection appears to have been interrupted",
```

The SDK exposes `connection_status` only in the `lbrynet status` command, an example below:
```
(lbry-venv) patrick@illuminati:~/Development/lbry/lbry-sdk$ lbrynet status | grep -B5 -A5 connection_status
      "total_received": 0,
      "total_sent": 0
    },
    "finished_blobs": 0
  },
  "connection_status": {
    "code": "connected",
    "message": "No connection problems detected"
  },
  "dht": {
    "node_id": "91c38fff9b41a60cb503a291da815fdd6a33e21e73845a74924482743b0169cdb53204243ba4b9f56243cef59ccea38a",
```

Resolving this issue is a change to `daemon.py` to enumerate these values, and regeneration of `docs/api.json`.  I cannot assign this to myself, but I will make the changes described above and link a PR for the same.

**Comment by patrickkeane** [comment]
This issue is resolved by #3208 -- though I'm not exactly certain of the process to update api.json -- it seemed to be significantly changed after rerunning `generate_json_api.py`.  If that is expected, I can attach the updated file.

The enumerated values are not well explained, but at least the possible values are now itemized following a similar pattern for values of `lbrynet version`'s `build` key.

**Comment by patrickkeane** [comment]
For what its worth, to answer the other issue mentioned in the issue description, the format for language in the `publish` command has been documented since this issue was opened:
```
            --languages=<languages>        : (list) languages used by the channel,
                                                    using RFC 5646 format, eg:
                                                    for English `--languages=en`
                                                    for Spanish (Spain) `--languages=es-ES`
                                                    for Spanish (Mexican) `--languages=es-MX`
                                                    for Chinese (Simplified) `--languages=zh-Hans`
                                                    for Chinese (Traditional) `--languages=zh-Hant`
```
